src/EnergyPlus/VariableSpeedCoils.cc

```diff
@@ -242,6 +246,168 @@ namespace VariableSpeedCoils {
 		VarSpeedCoil.deallocate();
 	}
 
+	// Default Constructor
```

Comment:
Hmmmmm default constructor in the cc file.  There's nothing wrong with this.  However, in the entirety of the code base we have defined the default constructors in the header in the struct definition.  Since we don't have a policy, could you match the rest of the codebase and simply put this over in the struct itself.  Shouldn't be anything but a cut/paste and removing the `VariableSpeedCoilData::` scope declaration.